cluster/test-deploy/Makefile

```diff
@@ -16,8 +18,8 @@ sh: check-cluster
 .PHONY: sh
 
 check-cluster:
-ifndef WHAT
-	$(error Must set WHAT= to a unique cluster name)
+ifndef INSTANCE_PREFIX
```

Comment:
I don't want to do this here, it's inconsistent with other playbooks.  We should always set INSTANCE_PREFIX to WHAT.

Reply:
Reverted this change.